### ISSUE TITLE
[@draft-js-plugins/emoji] - Testing is broken on react-testing-library

### ISSUE THREAD

**Issue by fdemian** [draft-js-plugins/draft-js-plugins#2704]
## Environment

I'm using draft-js plugins inside `create-react-app`

Reelevant packages: 

```
@draft-js-plugins/editor: ^4.1.2
@draft-js-plugins/emoji: ^4.6.0
@testing-library/jest-dom: "^5.16.4,
@testing-library/react: 13.0.0
react:^18.0.0
react-dom:^18.0.0
```

- @draft-js-plugins/editor version: 4.1.2
- plugin name and version: emoji plugin, version 4..6.0

## Description

While the emoji plugin is working correctly, it seems to break during testing. Running tests with react-testing-library throws the following error:

```TypeError: Cannot read properties of undefined (reading 'joypixels')```

## Reproducible Demo

Below is a minimalistic example that I'm using to test the plugins:

App.js
```
import React, { useState } from 'react';
import Editor, { createEditorStateWithText } from '@draft-js-plugins/editor';
import createEmojiPlugin from '@draft-js-plugins/emoji';

const emojiPlugin = createEmojiPlugin();
const { EmojiSuggestions, EmojiSelect } = emojiPlugin;
const initialState = createEditorStateWithText("....");

const styles = {
  height: '500px',
  width: '80%',
  marginLeft: '10%',
  border: '1px solid black',
  marginTop: '50px'
}

const App = () => {

  const [state, setState] = useState(initialState);

  return(
  <div className="App">
    <div style={styles}>
      <Editor
        editorState={state}
        onChange={setState}
        plugins={[emojiPlugin]}
      />
      <EmojiSuggestions />
    </div>
  </div>
  );

}

export default App;
```

App.test.js
```
import { render, screen } from '@testing-library/react';
import App from './App';

test('<Editor /> / Render correctly.', async () => {
 render(<App />);
 expect(screen.getByRole("combobox")).toBeInTheDocument();
});
```

This is the error being thrown by react-testing-library in full:

 ```
FAIL  src/App.test.js
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'joypixels')

      1 | import React, { useState } from 'react';
      2 | import Editor, { createEditorStateWithText } from '@draft-js-plugins/editor';
    > 3 | import createEmojiPlugin from '@draft-js-plugins/emoji';
        | ^
      4 |
      5 | const emojiPlugin = createEmojiPlugin();
      6 | const { EmojiSuggestions, EmojiSelect } = emojiPlugin;

      at Object.<anonymous> (node_modules/emoji-toolkit/lib/js/joypixels.js:502:25)
      at Object.<anonymous> (node_modules/@draft-js-plugins/emoji/lib/index.cjs.js:10:20)
      at Object.<anonymous> (src/App.js:3:1)
      at Object.<anonymous> (src/App.test.js:2:1)
      at TestScheduler.scheduleTests (node_modules/@jest/core/build/TestScheduler.js:333:13)
      at runJest (node_modules/@jest/core/build/runJest.js:404:19)
```

**Comment by fxOne** [comment]
There is no `joypixels` in this library. Can you provide a minimalist repository to reproduce the issue?

**Comment by fdemian** [comment]
Ok, the problem was on my end, but I'll leave this comment here in case anyone else comes across this.

Turns out that I was using Draft editor plugins inside of a component that used ant.design and jest can't propperly handle import files (hence, It was apparently mishandling the emoji plugin files). 

Once I followed the instructions here: 

https://ant.design/docs/react/use-with-create-react-app#Test-with-Jest

Jest no logner throws an error.

Sory for the noise!
